gallery.css

```diff
@@ -0,0 +1,95 @@
+@import url("https://fonts.googleapis.com/css2?family=Asap&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Poppins:ital,wght@0,300;1,600&family=Yusei+Magic&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #2e2b28;
+  font-size: calc(0.75em + 1vmin);
+}
+
+header {
+  display: flex;
+  width: 100%;
+  margin: auto;
+  align-items: center;
+}
+
+nav {
+  flex: 2;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+}
+
+.nav-link {
+  font-size: 1.2em;
+  text-decoration: none;
+  color: #e9efed;
+  font-family: "Asap", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 5px;
+  justify-content: center;
+}
+
+.box {
+  position: relative;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 0.5em;
+}
+
+.box img {
+  width: 100%;
+  height: 100%;
+  box-shadow: 0px 5px 5px orange;
+  padding: 5px;
+}
+
+.hobbies {
+  align-items: center;
+  margin: 3em;
+  padding: 3em;
+}
+
+h1 {
+  font-family: "Montserrat", sans-serif;
+  color: orange;
+  font-size: 3em;
+  font-weight: 400;
+}
+
+@media screen and (min-width: 200px) {
```

Comment:
so you're doing mobile first design here?  and what device is less than 200px?  That is a very small screen size nowadays!

Reply:
I would recommend you do max-width because you're designing with the desktop in mind.